eng/code-quality-reports/src/main/resources/revapi/revapi.json

```diff
@@ -430,6 +430,16 @@
         "code": "java.method.defaultMethodAddedToInterface",
         "new": "method byte[] com.azure.core.util.serializer.SerializerAdapter::serializeToBytes(java.lang.Object, com.azure.core.util.serializer.SerializerEncoding) throws java.io.IOException",
         "justification": "Method exposed from Azure Core"
+      },
+      {
+        "code": "java.method.added",
+        "new": "method void com.azure.core.http.rest.PagedFlux<T>::<init>(java.util.function.Function<java.lang.Integer, reactor.core.publisher.Mono<com.azure.core.http.rest.PagedResponse<T>>>, java.util.function.BiFunction<java.lang.String, java.lang.Integer, reactor.core.publisher.Mono<com.azure.core.http.rest.PagedResponse<T>>>)",
+        "justification": "Search doesn't expose this type from Azure Core"
+      },
+      {
+        "code": "java.method.added",
+        "new": "method void com.azure.core.http.rest.PagedFlux<T>::<init>(java.util.function.Function<java.lang.Integer, reactor.core.publisher.Mono<com.azure.core.http.rest.PagedResponse<T>>>)",
+        "justification": "Search doesn't expose this type from Azure Core"
```

Comment:
Are these because search is releasing a patch version instead of a minor version? If so, we should consider releasing it as a minor version update.